sdk/containerservice/arm-containerservice/review/arm-containerservice.api.md

```diff
@@ -140,6 +158,8 @@ export type AgentPoolsAbortLatestOperationResponse = AgentPoolsAbortLatestOperat
 
 // @public
 export interface AgentPoolsCreateOrUpdateOptionalParams extends coreClient.OperationOptions {
+    ifMatch?: string;
```

Comment:
header changes.

Reply:
it's an api level breaking see
https://github.com/Azure/azure-rest-api-specs/pull/30901/checks?check_run_id=31923669531